CHANGELOG.md

```diff
@@ -2,16 +2,29 @@
 
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## Next Release
+
 ### Added
 - Support for a new configuration language: JSON. You can now write
   JSON semgrep patterns with -lang json
 - Support for '...' inside set and dictionaries
 - Version check to recommend updating when out-of-date, disable with `--disable-version-check`
+- Support for multiline pattern-where-python
+- `--dry-run` flag to show result of autofixes without modifying any files
```

Comment:
Is it `--dry-run` or `--dryrun`? I think I made it `--dryrun` when I add it.

Reply:
ooo you right